packages/block-editor/src/components/inserter/hooks/use-patterns-state.js

```diff
@@ -21,8 +21,8 @@ import { __, sprintf } from '@wordpress/i18n';
 const usePatternsState = ( onInsert ) => {
 	const { patternCategories, patterns } = useSelect( ( select ) => {
 		const {
-			__experimentalBlockPatterns,
-			__experimentalBlockPatternCategories,
+			__experimentalBlockPatterns = [],
+			__experimentalBlockPatternCategories = [],
```

Comment:
There's a file called `defaults.js` in the "store" folder of block-editor that is meant to be used for these initializations. Maybe we should use it instead to avoid the need for default values in all places where we use these?